src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,26 @@
+
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import com.univocity.parsers.csv.CsvWriter
+import com.univocity.parsers.csv.CsvWriterSettings
+
+fun main(args: Array<String>) {
+
+
+    private fun readCarFromCsv(fileName: String): Car {
```

Comment:
This function should not be declared _within_ the `main` function. Please move it.

Also, it says that it returns a single `Car`. But wouldn't it make more sense to return an entire list of cars?